include/robot_localization/ros_filter.h

```diff
@@ -648,6 +648,10 @@ template<class T> class RosFilter
     //!
     geometry_msgs::TransformStamped worldBaseLinkTransMsg_;
 
+    //! @brief The time of the most recent published state
```

Comment:
In https://github.com/cra-ros-pkg/robot_localization/pull/595/files the `permit_corrected_publication` ROS parameter and `permitCorrectedPublication_` variables were created. Did you miss that part, or you simply don't want that part?

Reply:
I just left it out since I don't think we would ever use those params and I was trying to keep it as minimal as possible.

I think we need to sync fully with upstream at some point so we can reintegrate properly then.